.github/workflows/ci.yaml

```diff
@@ -964,6 +964,109 @@ jobs:
         run: |
           ./script/check_dirty
 
+  pytest-postgres:
```

Comment:
This job should be added the the `needs` of the `coverage` job. This ensures the coverage job waits for this `pytest-postgres` job to finish (missed that in the previous PR review, sorry 🙏 )